src/components/PokeList/PokeList.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function PokeList({ pokemon }) {
+  return (
+    <div>
+      {pokemon.map((pokes) => (
```

Comment:
`poke `might make more sense here as this is referring to one individual Pokémon object